arboretum/kubernetes/reports/compliance_oscal_observations.py

```diff
@@ -0,0 +1,227 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+The compliance OSCAL observations report.
+
+A json report comprising NIST OSCAL Assessment Results Observations generated
+by processing compliance operator fetcher cluster_resource evidence. The
+embedded XML within the cluster_resource evidence is transformed to produce the
+report. If an optional oscal_metadata file is specified, then the report is
```

Comment:
```suggestion
report. If an optional OSCAL metadata file is specified, then the report is
```

...again this is confusing.  Is this a file or evidence gathered from the locker?

Reply:
Changed "oscal_metadata" to "OSCAL metadata". The metadata (if present) makes up for what is lacking in the cluster_resource evidence to produce a better OSCAL. As mentioned in a comment above, the metadata should change over time as the resources contained within the cluster change.